docs/en/advanced_guides/how_to.md

```diff
@@ -196,3 +196,29 @@ Output of the above script is as below:
 ```
 
 Users can get the channels of the new backbone by Replacing the `ResNet(depth=18)` in this script with their customized backbone.
+
+# Use Detectron2 Model in MMDetection
+
+Users can use `Detectron2Wrapper` to use Detectron2's model in MMDetection. We give [Faster R-CNN](../../../configs/misc/d2_faster-rcnn_r50-caffe_fpn_ms-90k_coco.py),
```

Comment:
sers can use `Detectron2Wrapper` to use Detectron2's model -> sers can use `Detectron2Wrapper` to run Detectron2's model
we give -> we provide examples of xxx in MMDetection.